tests/backup/backup_helper.go

```diff
@@ -2180,7 +2181,7 @@ func upgradeStorkVersion(storkImageToUpgrade string) error {
 	if err != nil {
 		return err
 	}
-	err = apps.Instance().ValidateDeployment(updatedStorkDeployment, k8s.DefaultTimeout, k8s.DefaultRetryInterval)
+	err = apps.Instance().ValidateDeployment(updatedStorkDeployment, podReadyTimeout, podReadyRetryTime)
```

Comment:
Why do we have to wait for 30 min for the stork pods to be up and running? I dont think we should wait that long right?

Reply:
We can make this 15 mins

Reply:
made it to 15 mins.